{"uuid": "ccaa58b5-6f46-455e-ac19-1a049acf4e5d", "children": ["65c94237-f63f-447b-b618-834952f60542"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "ValueError: response body:\n{\"message\":\"API rate limit exceeded for 136.226.252.188. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)\",\"documentation_url\":\"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting\"}\n\nrequest url:\nhttps://api.github.com/repos/mozilla/geckodriver/releases/latest\nresponse headers:\n{'Date': 'Thu, 23 Oct 2025 07:09:43 GMT', 'Server': 'Varnish', 'Strict-Transport-Security': 'max-age=31536000; includeSubdomains; preload', 'X-Content-Type-Options': 'nosniff', 'X-Frame-Options': 'deny', 'X-XSS-Protection': '1; mode=block', 'Content-Security-Policy': \"default-src 'none'; style-src 'unsafe-inline'\", 'Access-Control-Allow-Origin': '*', 'Access-Control-Expose-Headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-RateLimit-Used, X-RateLimit-Resource, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset', 'Content-Type': 'application/json; charset=utf-8', 'Referrer-Policy': 'origin-when-cross-origin, strict-origin-when-cross-origin', 'X-GitHub-Media-Type': 'github.v3; format=json', 'X-RateLimit-Limit': '60', 'X-RateLimit-Remaining': '0', 'X-RateLimit-Reset': '1761203401', 'X-RateLimit-Resource': 'core', 'X-RateLimit-Used': '60', 'Content-Length': '281', 'X-GitHub-Request-Id': '34FE:2E75C9:D76652:1147928:68F9D4B7'}\n\n", "trace": "  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\AgileBanking\\conftest.py\", line 108, in driver\n    driver = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))\n                                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\webdriver_manager\\firefox.py\", line 39, in install\n    driver_path = self._get_driver_binary_path(self.driver)\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\webdriver_manager\\core\\manager.py\", line 40, in _get_driver_binary_path\n    file = self._download_manager.download_file(driver.get_driver_download_url(os_type))\n                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\webdriver_manager\\drivers\\firefox.py\", line 37, in get_driver_download_url\n    driver_version_to_download = self.get_driver_version_to_download()\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\webdriver_manager\\core\\driver.py\", line 48, in get_driver_version_to_download\n    return self.get_latest_release_version()\n           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\webdriver_manager\\drivers\\firefox.py\", line 29, in get_latest_release_version\n    resp = self._http_client.get(\n        url=self.latest_release_url,\n        headers=self.auth_header\n    )\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\webdriver_manager\\core\\http.py\", line 36, in get\n    self.validate_response(resp)\n    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^\n  File \"C:\\Users\\Ascendion\\PycharmProjects\\PythonProject1\\.venv\\Lib\\site-packages\\webdriver_manager\\core\\http.py\", line 19, in validate_response\n    raise ValueError(\n    ...<3 lines>...\n    )\n"}, "start": 1761203384941, "stop": 1761203393041}], "start": 1761203384941, "stop": 1761203393093}